### PR TITLE
lmdb: add `lmdb.pc`

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Needed for #130508.

### CODE REVIEW

File: Formula/lmdb.rb [hunk 1]
```diff
@@ -22,16 +22,39 @@ class Lmdb < Formula
...
+    <<~EOS
+      prefix=#{opt_prefix}
+      exec_prefix=${prefix}
+      libdir=${prefix}/lib
+      includedir=${prefix}/include
+
+      Name: lmdb
+      Description: #{desc}
+      URL: #{homepage}
+      Version: @PACKAGE_VERSION@
```
Comment (carlocab) [inline Formula/lmdb.rb:48]:
```suggestion
      Version: #{version}
```

**Comment by ZhongRuoyu** [comment]
Ok, here's another problem: `lmdb-sys`'s `build.rs` seems to be calling `pkg-config liblmdb`, not `lmdb`: https://github.com/danburkert/lmdb-rs/blob/c64f0b4f8280190525a6430388e714739f18976b/lmdb-sys/build.rs#L13

Should we add a symlink here? Or we could also add a workaround on `grin`'s side.

**Comment by carlocab** [comment]
Just add a symlink I think.

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
ARM64 dependent tests seems stuck. This shouldn't affect any dependents, so I think we're good to merge this.

**Review by carlocab — APPROVED** [review]

**Pull request merged**